### ISSUE TITLE
Put content on the skeleton Landing Page

### ISSUE THREAD

**Issue by catherineunciano** [IT3A-2016/Paper-Icebergs#34]
Ask @giomigueltada  and @Nhelie  about the contents.

**Comment by catherineunciano** [comment]
<!DOCTYPE html>
<html lang="en-US">
<head>

	<title>QWS</title>
	
	<meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1, description" content="">
	<meta name="author" content="">
	
	<link rel="stylesheet" href="../bootstrap/css/bootstrap.min.css" type="text/css">
	<link rel="stylesheet" href="../bootstrap/css/fonts.css" type="text/css">
	<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
	
	<script src="../smoothscroll.js"></script>
	<script src="../webscript.js"></script>
	<script src="../js/jquery.js"></script>
	<script src="../js/bootstrap.min.js"></script>
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
	<script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
	p {font-family: 'Montserrat', sans-serif;font-size:20px;}
	
</head>
<body class="cusbodycolor" oncontextmenu="return false">
	
	<div id="wrapper">
		<div class="row" style="background-color:#2471A3;"><br></div>
		<div class="row" style="background-color:#2471A3;"><br></div>
		<div class="row" style="background-color:#2471A3;"><br></div>
		<div class="col-lg-4"></div>
        <div class="col-lg-12 col-md-12 col-sm-12">
			<div class="row">
				<div class="col-lg-12"  style="padding:10px 0 0 0;text-align:center;">
					<div class="col-lg-12">
						<div class="col-lg-12">
							<div class="col-lg-12">
								<div class="col-lg-12">
									<div class="col-sm-9"></div>
									<div class="col-sm-1">
										<div style="">
											<a class="nav-item nav-link" href="login.php"><p style="color:#000000;">Login</p></a>
										</div>
									</div>
									<div class="col-sm-1">
										<div style="">
											<a class="nav-item nav-link" href="signup.php"><p style="color:#000000;">Signup</p></a>
										</div>	
									</div>
								</div>	
							</div>	
						</div>	
					</div>
				</div>  
			</div>
		</div>
		<div class="row"><br></div>
		<div class="row"><br></div>
		<div class="row"><br></div>
		<div class="col-lg-12 col-md-12 col-sm-12"> 
			<div class="col-lg-12 col-md-12 col-sm-3">
				<div class="row">
					<div class="col-lg-12 col-md-12 col-sm-12">
						<div class="col-lg-12 col-md-12 col-sm-12">
							<div class="col-lg-12 col-md-12 col-sm-12">
								<div class="col-lg-12 col-md-12 col-sm-12" style="text-align:center;padding:100px 0 50px 0;background-color:#3498DB;">
									<img src="../img/logo.png" alt="Paper-Icebergs Logo" height="150" width="150" align="center"></img>
									<p><br><br>Automated Queueing Web-based System</p>
								</div>
							</div>
						</div>
					</div>
				</div>
				<div class="row"><br></div>
				<div class="row">
					<div class="col-lg-12 col-md-12 col-sm-12">
						<div class="col-lg-12 col-md-12 col-sm-12">
							<div class="col-lg-12 col-md-12 col-sm-12">
								<p>
									<div class="col-lg-12 col-md-12 col-sm-12"  style="text-align:center;padding:20px 0 100px 0;background-color:#ECF0F1;">
										About the System<br><br>
										<div class="col-lg-12 col-md-12 col-sm-12"  style="text-align:center;padding:20px 0 20px 0;background-color:#ECF0F1;">
											The system is composed of web-based part and the mobile app part. The web-based is for <br>
											the one who renders services (e.g. cashiers) while the mobile app is for the customers’ use. <br>
											The mobile app won’t be much beneficial to be installed by the customer since the mobile app <br>
											will require user’s login account and this is provided by the company who will use <br>
											the web-based system. 
										</div>
									</div>
								</p>
							</div>
						</div>
					</div>
				</div>
				<div class="row"><br></div>
				<div class="row">
					<div class="col-lg-12 col-md-12 col-sm-12">
						<div class="col-lg-12 col-md-12 col-sm-12">
							<div class="col-lg-12 col-md-12 col-sm-12">
								<p>
									<div class="col-lg-12 col-md-12 col-sm-12" style="text-align:center;padding:20px 0 100px 0;background-color:#3498DB;">
										How It Works<br><br>
										<div class="col-lg-12 col-md-12 col-sm-12"  style="text-align:center;padding:20px 0 20px 0;background-color:#3498DB;">
											The customers will walk-in and get their number in the queue by filling up the information<br>
											required in the mobile app to make reservation of that number for the customer's turn and for <br>
											the customer's benefit of ease and time-free that the system will notify it's customers before<br>
											their due time.
										</div>
									</div>
								</p>
							</div>
						</div>
					</div>
				</div>
				<div class="row"><br></div>
				<div class="row">
					<div class="col-lg-12 col-md-12 col-sm-12">
						<div class="col-lg-12 col-md-12 col-sm-12">
							<div class="col-lg-12 col-md-12 col-sm-12">
								<div class="col-lg-12 col-md-12 col-sm-12" style="text-align:center;padding:20px 0 50px 0;background-color:#ECF0F1;">
									<hr><br><br>
									<div class="col-lg-12 col-md-12 col-sm-12"  style="text-align:center;padding:20px 0 20px 0;background-color:#ECF0F1;">
										<span style="color:#000000;"><a class="nav-item nav-link" href="about.php"> About Us</a> | </span>
										<span style="color:#000000;"><a class="nav-item nav-link" href="faq.php"> FAQ </a> | </span>
										<span style="color:#000000;"><a class="nav-item nav-link" href="contact.php"> Contact Us </a></span>
									</div>
									<div class="row"></div>
									<div class="col-lg-12 col-md-12 col-sm-12"  style="text-align:center;padding:20px 0 20px 0;background-color:#ECF0F1;">
										<a class="nav-item nav-link" href="#"><img src="../img/fb.png" alt="facebook-icon" height="50" width="50"></img></a>
										<a class="nav-item nav-link" href="#"><img src="../img/twitter.png" alt="twitter-icon" height="50" width="50"></img></a>
										<a class="nav-item nav-link" href="#"><img src="../img/ig.png" alt="instagram-icon" height="50" width="50"></img></a>
									</div>
								</div>
							</div>
						</div>
					</div>
				</div>
			</div>
		</div>
	</div>
	
	<div class="row" style="padding:20px 0 20px 0;"><br><br></div>
	<footer style="text-align:center;background-color:#707B7C;color:#000000;">Paper-Icebergs 2017 &copy; All rights reserved.</footer>
	
</body>
</html>
